src/notation/internal/notationinteraction.cpp

```diff
@@ -525,9 +523,7 @@ void NotationInteraction::selectTopOrBottomOfChord(MoveDirection d)
         return;
     }
 
-    score()->select(target, SelectType::SINGLE);
-
-    notifyAboutSelectionChanged();
+    select({ target }, SelectType::SINGLE);
 }
```

Comment:
...this is gonna cause a LOT of conflicts with the work on my PR that nobody's looked at yet...